tests/integration/auth/client/cli/suite_test.go

```
@@ -85,6 +85,9 @@ func (s *CLITestSuite) SetupSuite() {
 	s.Require().NoError(err)
 	s.val, err = valAcc.GetAddress()
 	s.Require().NoError(err)
+	pub, err := valAcc.GetPubKey()
+	s.Require().NoError(err)
+	kb.SaveOfflineKey("newAccount", pub)
```

golangci-lint (1.62.2)

Error return value of `kb.SaveOfflineKey` is not checked

(errcheck)